packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model/hooks.esm.t

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
by generating hooks in the document model instead of in editors, we can avoid duplication while also making sure these hooks can be used by any drive or document editor